lib/raven/integrations/delayed_job.rb

```diff
@@ -1,8 +1,12 @@
 require 'delayed_job'
 
+require 'raven/context_filter'
+
 module Delayed
   module Plugins
     class Raven < ::Delayed::Plugin
+      include Raven::ContextFilter
```

Comment:
If you want to call `Raven::ContextFilter.filter` directly, this include is unnecessary